### ISSUE TITLE
Please make the icon more convenient

### ISSUE THREAD

**Issue by chaoshuaihaohao** [schorsch1976/AsciiFlowQT#2]
This program is very good, I use asciiflow is also very uncomfortable, and asciiflow can't export complete text, so I use it very inconvenient. But the blogger's program is also some imperfect, the icon is not very intuitive, the line is not very beautiful. I hope it can be more convenient and practical. Recently I saw the monodraw software of MacOS, it seems very good, but I use the Linux system,,, very sad, No good ASCII drawing software

**Comment by asmwarrior** [comment]
I also agree with the OP's suggestion, the icon of this project should be improved.

If you would like to see the screen shot of the main frame and the icons, you can see this discussion, I have post some screen shot images there:

[This project can be built with the wxWidget library?](https://github.com/schorsch1976/AsciiFlowQT/issues/4)

Generally, I think some free/open icon can be used, there are many web site to allow us to download those kinds of icons.

**Comment by asmwarrior** [comment]
[New File Vector SVG Icon (10) - SVG Repo](https://www.svgrepo.com/svg/361247/new-file)

I use the icon from the above link, and I first download the png file, which is 256*256 pixel size, than I open the GIMP, and convert this png to xpm file.

Now, here is the result application.

![image](https://github.com/schorsch1976/AsciiFlowQT/assets/561818/3f183d87-527a-4246-91b6-ceb449600397)

So, this is the direction to improve.

But I see when compiling, I got some gcc warning:

~~~~
F:\code\AsciiFlowQT\src\Common\include/AsciiFlowCommon/ico/New.png.xpm:3:1: warning: ISO C++ forbids converting a string constant to 'char*' [-Wwrite-strings]
~~~~

It looks like the xpm file has some issue? But when I export the xpm file in GIMP, I don't see any option I can tweak. Only one option is the transparent threshold.

**Comment by asmwarrior** [comment]
OK, I think we have to edit the xpm file, and add the `const`.

See discussion below: [Warning in C++17 when including XPM image](https://stackoverflow.com/questions/61755276/warning-in-c17-when-including-xpm-image), though this project needs C++ 20.
